src/component/parent/drivers.js

```diff
@@ -150,6 +150,7 @@ export let RENDER_DRIVERS = {
 
             this.iframe.style.zIndex   = MAX_Z_INDEX;
             this.iframe.style.position = 'fixed';
+            this.iframe.className  = 'xcomponent-lightbox bounceInUp';
```

Comment:
So I have renamed the class to xcomponent-lightbox-open. Using just xcomponent-lightbox and naming the animation: bounceInUp in it was creating some issues(overlay wasn't created and the box got cut off). Nonetheless I hope it resolves the bounceInUp hard coding issue.